private/README.md

```diff
@@ -0,0 +1,3 @@
+﻿# Private Functions
```

Comment:
Can we explain why we don't use Verb-Noun format with private functions?

Reply:
Done.